### DISCUSSION TITLE
How to set 'ACEc'?

### DISCUSSION THREAD

**Discussion by sghqsads** [CURENT/andes#477] [Q&A] [answered]
Hello everyone,

In terms of using 'ACEc', I was wondering whether we need to:
1. must set all bus of the system in input table 'ACEc'?
or
2. can set one bus(this bus must be equipped with BusFreq) of the system in input table 'ACEc'?
or
3.can set one bus(equipped with BusFreq is not necessary ) of the system in input table 'ACEc'?

Thank you!

**Comment by cuihantao** [comment, marked as answer]
Here is an example: https://github.com/CURENT/andes/blob/master/andes/cases/ieee14/ieee14_ace.xlsx

You only need create `ACEc` entries for the desired buses, create `BusFreq` entries pointing to the same buses, and set the proper `busf` for `ACEc` entries.

**Reply by sghqsads** [reply]
Thank you!
